scheduler/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16.3 as build
```

Comment:
Can you pin this as well? see https://github.com/ossf/package-analysis/blob/main/analysis/Dockerfile#L1 for an example